### ISSUE TITLE
JsonFormatter

### ISSUE THREAD

**Issue by tommywo** [cucumber/microcuke#6]
Is JsonFormatter in scope?

**Comment by aslakhellesoy** [comment]
Yes, but it will output a different format than the existing json formatters. We're aiming for a simpler format with only results (no ast info, just pointers to source files and lines)
